examples/densenet_pytorch/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:16.04
```

Comment:
Recommend not having this Dockerfile. If you want to showcase using Containers, build on top of a base MMS container and add just the pytorch and other required libraries. The README only seems to showcase running mms natively.